### ISSUE TITLE
Plot the GAN results

### ISSUE THREAD

**Issue by YakubFahimLuckyarno** [Kitsunetic/head-project#4]
Hi, 

it seems that the run_gan_training.py does not show the figures yet. So, how should I do that? Can I plot it using the same way in the run_training.py?

**Comment by Kitsunetic** [comment]
Yes

Line 102 ~ 139 of run_training.py are plotting codes.  
You can use this codes by changing your input shapes.

**Comment by YakubFahimLuckyarno** [comment]
> Yes
> 
> Line 102 ~ 139 of run_training.py are plotting codes.
> You can use this codes by changing your input shapes.

Okay. It seems the one that I need is in the line 201 ~ 239 of run_training.py. I will try this one. 

Another question is about the element of MEANS and STDS. For example, the MEANS contains 6 values. For each values, what is it? Is it correspond to certain parameter, i.e. yaw or pitch or roll or acceleration_x, etc.

**Comment by Kitsunetic** [comment]
Yes MEANS and STDS are mean and standard deviation of my dataset calculated outside the code for data standardization.
It's order is [yaw, pitch, roll, acceleration_x, y, z].
